File: Python/sequenciais/duracao_em_segundos.py
# Exercício: Duração em segundos
# Objetivo: converter uma duração em segundos para horas, minutos e segundos.

duracao = int(input("Digite a duracao em segundos: "))

horas = duracao // 3600
resto = duracao % 3600

minutos = resto // 60
segundos = resto % 60

print(f"{horas:02d}:{minutos:02d}:{segundos:02d}")

File: Java/Estruturas Sequenciais/duracao_em_segundos.java
// Exercício: Duração em segundos
// Objetivo: converter uma duração em segundos para horas, minutos e segundos.

import java.util.Locale;
import java.util.Scanner;

public class duracao_em_segundos {
    public static void main(String[] args) {
        
        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        int duracao, horas, minutos, segundos, resto;

        System.out.print("Digite a duracao em segundos: ");
        duracao = sc.nextInt();

        horas = duracao / 3600;
        resto = duracao % 3600;

        minutos = resto / 60;
        segundos = resto % 60;

        System.out.println(horas + ":" + minutos + ":" + segundos);

        sc.close();
    }
}
